bin/prettier.js

```diff
@@ -14,7 +14,10 @@ const argv = minimist(process.argv.slice(2), {
     "single-quote",
     "trailing-comma",
     "bracket-spacing",
-    // See https://github.com/chalk/supports-color/#info
+    // The supports-color package (a sub sub dependency) looks directly at
+    // `process.argv` for `--no-color` and such-like options. The reason it is
+    // listed here is to avoid "Ignored unknown option: --no-color" warnings.
+    // See https://github.com/chalk/supports-color/#info for more information.
```

Comment:
I suppose we can keep it, but it is a little weird for a dependency for be reading straight from `process.argv`. I wish it didn't do that. But it's fine to document the default.